src/core/style/_variables.scss

```diff
@@ -24,7 +24,7 @@ $md-toggle-size: 20px !default;
 // TODO(jelbourn): all of these need to be revisited
 
 // The default animation curves used by material design.
-$md-linear-out-slow-in-timing-function: cubic-bezier(0.0, 0.0, 0.2, 0.1) !default;
```

Comment:
I'd prefer not to change this in this PR and instead do a review directly with the Material Design UX team.